tools/src/main/scala/scala/scalanative/build/Logger.scala

```diff
@@ -21,7 +21,7 @@ trait Logger {
 
   /** Logs at the debug level that the command `cmd` will start running. */
   def running(cmd: Seq[String]): Unit = {
-    val msg = "running" + nl + cmd.mkString(nl + "\t")
+    val msg = "Running" + nl + cmd.mkString(nl + "\t")
```

Comment:
Spurious change?

Reply:
No, this was intended as all the other logging messages start with a capital.